docs/commands/heap.md

```diff
@@ -27,10 +27,22 @@ gef➤ heap chunks <LOCATION>
 
 ![heap-chunks](https://i.imgur.com/2Ew2fA6.png)
 
+### `heap chunkrange` command ###
+
+Unlike 'heap chunks' this command tries (untill could read) to give information of all 
+Glibc chunks started from arbitrary location and less than another location. Increment is done only via heap size field.
```

Comment:
"started from arbitrary location and less then another location" could be simplified to "within a memory region".